shared/actions/platform.specific.native.js

```diff
@@ -1,5 +1,8 @@
 // @flow
 import * as PushNotifications from 'react-native-push-notification'
+import {PushNotificationIOS} from 'react-native'
+import * as PushConstants from '../constants/push'
+import {eventChannel} from 'redux-saga'
 
 function requestPushPermissions (): Promise<*> {
   return PushNotifications.requestPermissions()
```

Comment:
It's a requirement that the `PushNotifications` in `PushNotifications.requestPermissions()`. So `configure` (below) needs to be here.